tests/e2e/standalone/standalone_test.go

```diff
@@ -58,6 +58,63 @@ func TestStandaloneInstall(t *testing.T) {
 	}
 }
 
+func TestNegativeScenarios(t *testing.T) {
+	// Ensure a clean environment
+	uninstall()
+	daprPath := getDaprPath()
+
+	homeDir, err := os.UserHomeDir()
+	require.NoError(t, err, "expected no error on querying for os home dir")
+
+	t.Run("run without install", func(t *testing.T) {
+		output, err := spawn.Command(daprPath, "run", "test")
+		require.NoError(t, err, "expected no error status on run without install")
```

Comment:
I think a separate issue has to be created so that we properly exit with error on error from run. 
This needs to be separate task since the scenarios for run command are a bit more complex.

Reply:
Added https://github.com/dapr/cli/issues/684